flexget/plugins/output/slack.py

```diff
@@ -0,0 +1,120 @@
+from __future__ import unicode_literals, division, absolute_import
+import logging
+
+from flexget import plugin
+from flexget.event import event
+from flexget.utils import json
+from flexget.utils.template import RenderError
+
+log = logging.getLogger('slack')
+
+
+class OutputSlack(object):
+    """
+    Example:
+
+      slack:
+        webhook-url: <string>
+        [text: <string>] (default: "{{task}} - Download started: {{series_name}} {{series_id}}" -- accepts Jinja2)
+        [channel: <string>] (override channel, use "@username" or "#channel")
+        [username: <string>] (override username)
+        [icon-emoji: <string>] (override emoji icon, do not include the colons: e.g., use "tv" instead of ":tv:")
+
+    Configuration parameters are also supported from entries (e.g., through set).
+    """
+    default_body = ('{{task}} - Download started: '
+                    '{% if series_name is defined %}'
+                    '{{tvdb_series_name|d(series_name)}} {{series_id}} {{tvdb_ep_name|d('')}}'
+                    '{% elif imdb_name is defined %}'
+                    '{{imdb_name}} {{imdb_year}}'
+                    '{% else %}'
+                    '{{title}}'
+                    '{% endif %}')
+    schema = {
+        'type': 'object',
+        'properties': {
+            'webhook-url': {'type': 'string'},
+            'text': {'type': 'string', 'default': default_body},
+            'channel': {'type': 'string'},
+            'username': {'type': 'string'},
+            'icon-emoji': {'type': 'string'},
+        },
+        'required': ['webhook-url'],
+        'additionalProperties': False
+    }
+
+    @plugin.priority(0)
+    def on_task_output(self, task, config):
+
+        for entry in task.accepted:
+            webhook_url = config.get('webhook-url')
```

Comment:
I'm not familiar with slack api but wouldn't it be better to set a base url to use instead of passing the entire url from config?